homeassistant/config.py

```diff
@@ -86,7 +86,9 @@
 )
 
 DEFAULT_CONFIG = f"""
-# Configure a default setup of Home Assistant (frontend, api, etc)
+# Load a default set of integrations. This is a meta-component. It can be removed
+# however take care; you may strongly desire the 'frontend' integration, for example.
+# Documentation: https://www.home-assistant.io/integrations/default_config/
```

Comment:
We should not mention that it can be removed. 80% of the issues of a new feature not working is because people thought it would be a good idea to remove this.
```suggestion
# Loads default set of integrations. Do not remove.
```